migrations/1661437154069-clean-user-account.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require('mongoose');
+const { connect, close } = require('../src/utils/database');
+const { alert } = require('../src/logger');
+
+const splitToChunks = (array, parts) => {
+	const result = [];
+	// eslint-disable-next-line no-plusplus
+	for (let i = parts; i > 0; i--) {
+		const part = array.splice(0, Math.ceil(array.length / i));
+		if (part.length > 0) {
+			result.push(part);
+		}
+	}
+	return result;
+};
+
+// use your own name for your model, otherwise other migrations may fail.
+// The third parameter is the actually relevent one for what collection to write to.
+const User = mongoose.model('users220825', new mongoose.Schema({}), 'users');
+const Account = mongoose.model('accounts220825', new mongoose.Schema({}), 'accounts');
+
+// How to use more than one schema per collection on mongodb
+// https://stackoverflow.com/questions/14453864/use-more-than-one-schema-per-collection-on-mongodb
+
+// TODO npm run migration-persist and remove this line
+// TODO update seed data and remove this line
+
+module.exports = {
+	up: async function up() {
```

Comment:
you must be implemented both methods, up and down. If you can not rollback over down add a logger.alert to with the information to it.